src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/InlinedTypes.cs

```diff
@@ -0,0 +1,247 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Microsoft.Interop
+{
+    internal static class InlinedTypes
+    {
+        /// <summary>
+        /// The ClassDeclarationSyntax for the following
+        /// <code>
+        /// file abstract unsafe class IUnknownVTableComWrappers : ComWrappers
+        /// {
+        ///     public static void GetIUnknownImpl(out void* pQueryInterface, out void* pAddRef, out void* pRelease)
+        ///     {
+        ///         nint qi, addRef, release;
+        ///         ComWrappers.GetIUnknownImpl(out qi, out addRef, out release);
+        ///         pQueryInterface = (void*)qi;
+        ///         pAddRef = (void*)addRef;
+        ///         pRelease = (void*)release;
+        ///     }
+        /// }
+        /// </code>
+        /// </summary>
+        public static ClassDeclarationSyntax IUnknownVTableComWrappersInstance { get; } = GetIUnknownVTableComWrappersInstance();
```

Comment:
As follow up, we can update the public API (already approved) to expose `ComWrappers.GetIUnknownImpl` so we don't need to inline this.

Reply:
This isn't used anywhere yet, so if we don't need it for COM generator, I'll remove it.